qml/pages/ThreadView.qml

```diff
@@ -269,6 +269,24 @@ Page {
         for (var i=0;i<posts_length;i++) {
             var post = posts[i];
             var yours =  (loggedin.value == "-1") ? false : post.yours
+            var has_polls = !!post.polls  ? post.polls.length : 0
+            var polldata = []
+            if (i!=0) {
+                // can comments have polls?
```

Comment:
Yeah, every comment can also have multiple polls, so the repeater button approach is actually great for this, however unlikely scenario this is (there was a post in a middle of a thread with two polls, but like half a year ago so it's probably already closed, not sure how to find it)